Directory.Build.props

```diff
@@ -8,6 +8,8 @@
     <LangVersion>Latest</LangVersion>
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Do we need the documentation file?

Reply:
_Need_ is a strong word here. The editor config enforcement will error if documentation generation is disabled; that could be mitigated by disabling all of the editor config features that check the doc comments. I opt-ed to go the other direction to enable it across the board and fix all of the documentation problems.